AreaSetup[] area_setup_info
